README.md

```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
 ![Application-Architecture-Diagram](./docs/images/application-architecture-diagram/application_architecture_diagram.png)
 
 ## User Stories 
+We constructed user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying customers' unique needs and expectations, including a system administrator at Sakura company.
 
-Working in progress...
+### Initial Market Research
+1. As a busy parent, I want search functionality with a category filtering option to save time searching for the right product.
+
```

Comment:
Can you add more user stories for the below features:

- Password recovery
- Site navigation and responsive design
- Shopping cart: allow users to add products to cart, view items in cart and increase, decrease or remove items from cart.
- User functionality to view order history